apps/user-office-frontend-e2e/cypress/integration/templates_basic.ts

```diff
@@ -1378,40 +1413,98 @@ context('Template tests', () => {
         .find('[aria-label=Edit]')
         .first()
         .click();
+    });
+
+    it('Should render empty list if JSONPATH syntax is invalid', () => {
+      cy.intercept(
+        { method: 'GET', url: dynamicMultipleChoiceQuestion.url },
+        {
+          statusCode: 201,
+          body: dynamicMultipleChoiceQuestion.answers.arrayObject,
+        }
+      );
       cy.createDynamicMultipleChoiceQuestion(
         dynamicMultipleChoiceQuestion.title,
         {
           url: dynamicMultipleChoiceQuestion.url,
+          jsonPath: '$.[*].item',
           isMultipleSelect: true,
           firstTopic: true,
         }
       );
+
+      createProposalAndClickDropdownBehavior();
+
+      cy.get('[data-cy=dropdown-ul]').children().should('not.contain.value');
     });
-    it('Should be able to select options returned from external api', () => {
+
+    it('Should render empty list if external API is down', () => {
       cy.intercept(
         { method: 'GET', url: dynamicMultipleChoiceQuestion.url },
         {
-          statusCode: 201,
-          body: dynamicMultipleChoiceQuestion.answers,
+          statusCode: 404,
+          body: 'Not Found',
         }
       );
+      cy.createDynamicMultipleChoiceQuestion(
+        dynamicMultipleChoiceQuestion.title,
+        {
+          url: dynamicMultipleChoiceQuestion.url,
+          jsonPath: dynamicMultipleChoiceQuestion.jsonPath,
+          isMultipleSelect: true,
+          firstTopic: true,
+        }
+      );
+      createProposalAndClickDropdownBehavior();
 
-      cy.login('user1');
-      cy.visit('/');
-
-      cy.contains('New Proposal').click();
-      cy.get('[data-cy=call-list]').find('li:first-child').click();
+      cy.get('[data-cy=dropdown-ul]').children().should('not.contain.value');
+    });
 
-      cy.finishedLoading();
+    it('Should be able to use JSONPATH library to extract specific data from API response', () => {
+      cy.intercept(
```

Comment:
Since we now get values from the backend is it possible to get the options from there instead.

Reply:
I created mock server to fake the behavior in the new commit.